CMakeLists.txt

```diff
@@ -5,13 +5,14 @@ parseversion("src/rdkafka.h")
 
 project(RdKafka VERSION ${RDKAFKA_VERSION})
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/packaging/cmake/Modules/")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH}
```

Comment:
Avoid whitespace/indent changes unless necessary.

Reply:
There was inconsistency (two spaces, four spaces, tab characters) between CMakeLists.txt files, and even within the same file in the case of `src/CMakeLists.txt`, so I ran a pretty printer, `cmake-format` on them. I went back and removed any changes that weren't related to indentation though (it wanted to enforce 80 column limits for example)

Reply:
That's okay, but let's do reformatting in a separate PR

Reply:
Good point. I'll roll all this back. Makes the diff for this PR easier to read